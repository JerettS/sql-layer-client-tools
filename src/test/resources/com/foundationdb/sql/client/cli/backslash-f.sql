CREATE TABLE t ( label int, long_label int, a int);
INSERT INTO t (label, long_label, a) values (1,NULL,3), (4,NULL,6), (7,8,NULL);
SELECT * FROM t;

\f "LONGFIELDS"

SELECT * FROM t;

\f ","

SELECT * FROM t;

\x

SELECT * FROM t;
\f
SELECT * FROM t;

\x

SELECT * FROM t;


