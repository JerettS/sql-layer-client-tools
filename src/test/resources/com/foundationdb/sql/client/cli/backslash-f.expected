ROWS: 0

ROWS: 3

 label | long_label | a
-------+------------+---
     1 |            | 3
     4 |            | 6
     7 |          8 |
(3 rows)

Format separator as "LONGFIELDS"
 label LONGFIELDS long_label LONGFIELDS a
----------------+---------------------+------------
     1 LONGFIELDS            LONGFIELDS 3
     4 LONGFIELDS            LONGFIELDS 6
     7 LONGFIELDS          8 LONGFIELDS
(3 rows)

Format separator as ","
 label , long_label , a
-------+------------+---
     1 ,            , 3
     4 ,            , 6
     7 ,          8 ,
(3 rows)

Expanded display is on
-[ RECORD 1 ]
label      , 1
long_label ,
a          , 3
-[ RECORD 2 ]
label      , 4
long_label ,
a          , 6
-[ RECORD 3 ]
label      , 7
long_label , 8
a          ,

Format separator as ","
Format separator as "|"
-[ RECORD 1 ]
label      | 1
long_label |
a          | 3
-[ RECORD 2 ]
label      | 4
long_label |
a          | 6
-[ RECORD 3 ]
label      | 7
long_label | 8
a          |

Expanded display is off
 label | long_label | a
-------+------------+---
     1 |            | 3
     4 |            | 6
     7 |          8 |
(3 rows)